media-types.md

```diff
@@ -38,25 +38,38 @@ This section shows where the OCI Image Specification is compatible with formats
 
 **Similar/related schema**
 
-- [application/vnd.docker.distribution.manifest.list.v2+json](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list) - mediaType is different
+- [application/vnd.docker.distribution.manifest.list.v2+json](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list)
+  - `.mediaType`: only present on Docker
+  - `.annotations`: only present on OCI
+  - `.[]manifests.annotations`: only present on OCI
+  - `.[]manifests.platform`: ARM v6 is represented as `{"architecture":"arm","variant":"v6"}` on OCI, while represented as `{"variant:armv6l"}` on Docker
 
 ### application/vnd.oci.image.manifest.v1+json
 
 **Similar/related schema**
 
 - [application/vnd.docker.distribution.manifest.v2+json](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#image-manifest-field-descriptions)
+  - `.mediaType`: only present on Docker
+  - `.annotations`: only present on OCI
+  - `.config.annotations`: only present on OCI
+  - `.[]layers.annotations`: only present on OCI
 
 ### application/vnd.oci.image.layer.v1.tar+gzip
 
 **Interchangeable and fully compatible mime-types**
 
-- [application/vnd.docker.image.rootfs.diff.tar.gzip](https://github.com/docker/docker/blob/master/image/spec/v1.md#creating-an-image-filesystem-changeset)
+- [application/vnd.docker.image.rootfs.diff.tar.gzip](https://github.com/moby/moby/blob/v20.10.1/image/spec/v1.2.md#creating-an-image-filesystem-changeset)
 
 ### application/vnd.oci.image.config.v1+json
 
 **Similar/related schema**
 
-- [application/vnd.docker.container.image.v1+json](https://github.com/docker/docker/blob/master/image/spec/v1.md#image-json-description)
+- [application/vnd.docker.container.image.v1+json](https://github.com/moby/moby/blob/v20.10.1/image/spec/v1.2.md#image-json-description) (Docker Image Spec v1.2)
+  - `.config.Memory`: only present on Docker
+  - `.config.MemorySwap`: only present on Docker
+  - `.config.CpuShares`: only present on Docker
+  - `.config.Healthcheck`: only present on Docker
+  - `.config.Labels`: only present on OCI
```

Comment:
`.config.StopSignal` is not defined in `application/vnd.docker.container.image.v1+json`.

Reply:
updated, thanks

Reply:
Neither labels nor stopsignal are new OCI concepts so it is wrong to give an impression like they are OCI only. They were copied over from Docker but missing descriptions were not contributed back. The out-of-sync issue seems to be caused by the linked spec file reusing the `RunConfig` type from Docker API that is defined separately from image. We can just make sure the spec file lists all relevant fields so there is no confusion. `OnBuild`, `ArgsEscaped`, `Shell`, `StopTimeout` are also not defined in OCI. https://github.com/moby/buildkit/blob/master/frontend/dockerfile/dockerfile2llb/image.go#L32

Reply:
^ @AkihiroSuda

Reply:
Looks like the comment about "OnBuild, ArgsEscaped, Shell, StopTimeout" went unnoticed.